aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -201,6 +201,21 @@ class DispatchTable final {
      );
    }
 
+   KernelFunctionWrapper* getVariableWrapper(const Stack* stack) const {
+     if (variable_wrapper_ && dispatch_strategy_.get_is_variable(stack, operator_name_)) {
+       return variable_wrapper_;
+     }
+     return nullptr;
+   }
+
+   void registerVariableWrapper(KernelFunctionWrapper* wrapper) {
+     variable_wrapper_ = wrapper;
+   }
+
+   void deregisterVariableWrapper() {
```

Comment:
this isn't called from anywhere. You probably didn't implement deregistration correctly.